examples/vello_editor/src/text.rs

```diff
@@ -285,10 +315,23 @@ impl Editor {
                 let prev_pos = self.cursor_pos;
                 self.cursor_pos = (position.x as f32 - INSET, position.y as f32 - INSET);
                 // macOS seems to generate a spurious move after selecting word?
-                if self.pointer_down && prev_pos != self.cursor_pos {
+                if self.pointer_down && prev_pos != self.cursor_pos && !self.editor.is_composing() {
                     self.cursor_reset();
                     let cursor_pos = self.cursor_pos;
-                    self.transact(|txn| txn.extend_selection_to_point(cursor_pos.0, cursor_pos.1));
+                    self.drive(|drv| drv.extend_selection_to_point(cursor_pos.0, cursor_pos.1));
+                }
+            }
+            WindowEvent::Ime(Ime::Disabled) => {
+                self.drive(|drv| drv.clear_compose());
```

Comment:
Nit: These could also be `self.driver().clear_compose()`? Seems better not to introduce a closure here if not needed.

Reply:
Even better: I believe you could create one driver at the top of the function. And then everything else is just calling method(s) on that.